okta/services/idaas/resource_okta_api_token.go

```diff
@@ -0,0 +1,262 @@
+}
+
+func (r *apiTokenResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
+}
+
+type IPs struct {
+	IP types.String `tfsdk:"ip"`
+}
+
+type NetworkModel struct {
+	Connection types.String `tfsdk:"connection"`
+	Include    []IPs        `tfsdk:"include"`
+	Exclude    []IPs        `tfsdk:"exclude"`
+}
+
+type apiTokenResourceModel struct {
+	Id         types.String  `tfsdk:"id"`
+	Name       types.String  `tfsdk:"name"`
+	Network    *NetworkModel `tfsdk:"network"`
+	UserId     types.String  `tfsdk:"user_id"`
+	Created    types.String  `tfsdk:"created"`
+	ClientName types.String  `tfsdk:"client_name"`
+}
+
+func (r *apiTokenResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Required:    true,
+				Description: "The ID of the API token.",
+			},
+			"user_id": schema.StringAttribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "The ID of the user that the API token will be used.",
+			},
+			"name": schema.StringAttribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "The name of the API token.",
+			},
+			"created": schema.StringAttribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "Timestamp when the API token was created.",
+			},
+			"client_name": schema.StringAttribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "The name of the API token client",
+			},
+		},
+		Blocks: map[string]schema.Block{
+			"network": schema.SingleNestedBlock{
+				Description: "The Network Condition of the API Token.",
+				Attributes: map[string]schema.Attribute{
+					"connection": schema.StringAttribute{
+						Optional:    true,
+						Description: "The connection type of the Network Condition.",
+					},
+				},
+				Blocks: map[string]schema.Block{
+					"exclude": schema.SetNestedBlock{
+						NestedObject: schema.NestedBlockObject{
+							Attributes: map[string]schema.Attribute{
+								"ip": schema.StringAttribute{
+									Optional:    true,
+									Computed:    true,
+									Description: "The IP address the excluded zone.",
+								},
+							},
+						},
+					},
+					"include": schema.SetNestedBlock{
+						NestedObject: schema.NestedBlockObject{
+							Attributes: map[string]schema.Attribute{
+								"ip": schema.StringAttribute{
+									Optional:    true,
+									Computed:    true,
+									Description: "The IP address the included zone.",
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func (r *apiTokenResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	resp.Diagnostics.AddWarning(
+		"Create Not Supported",
+		"This resource cannot be created via Terraform.",
```

Comment:
The error could be better.
```suggestion
		"This is a read-only resource and cannot be created via Terraform. API tokens must be created outside of Terraform and imported for management.",
```